generator/generator.go

```diff
@@ -110,6 +110,16 @@ func execute(command string, force bool) error {
 		args = args[1:]
 	}
 
+	// Templates will often include a helm:generate header such as the following:
```

Comment:
do we still want the users to use helm:generate and not helmc:generate in their charts going forward?